contracts/mocks/extensions/tokenProperties/RMRKTokenPropertiesMock.sol

```diff
@@ -93,9 +89,22 @@ contract RMRKTokenProperties is IRMRKTokenProperties {
     * @param value The property value
     */
     function setStringProperty(uint256 tokenId, string memory key, string memory value) external {
-        _stringProperties[tokenId][key] = totalStringProperties;
-        stringProperties[totalStringProperties] = value;
-        totalStringProperties++;
+
```

Comment:
Again, based on what we agreed on, this needs:

```suggestion
if(reverseDictionary[value] == 0) {
    dictionary[dictionary.length + 1] = value;
    reverseDictionary[value] = dictionary.length;
}
```

I just assumed that value 0 is empty (like we do for asset & child IDs), so that this is smaller.